app.js

```diff
@@ -10,7 +10,7 @@ var readTorrent = require('read-torrent')
 var proc = require('child_process')
 var peerflix = require('./')
 var keypress = require('keypress')
-var open = require('open')
+var openUrl = require('open')
```

Comment:
Yeah, this is an ESLint bug, unfortunately. https://github.com/eslint/eslint/issues/892

It thinks we're overriding the global `window.open`, I think. :/ A workaround would be to remove the "no-undef" rule (detection of unused variables), but it's a really useful rule despite this bug...